new-client/package-lock.json

```diff
@@ -24177,7 +24232,6 @@
       "integrity": "sha512-hS4pxwn1ZGXVkmgD4tpFpaumUaAg2ZzbTrxltfC5yPw4BJV+mGkfnQOB4VpWEYZw2jv65Z0wLwDE/piQiPPZ3w==",
       "requires": {
         "@babel/runtime": "^7.6.0",
-        "@date-io/core": "1.x",
```

Comment:
When i run `npm install` in your branch the `"@date-io/core": "1.x"` is added again.. Not sure why this would be removed. Maybe try running `npm install` again.